dm/tests/others_integration_2.txt

```diff
@@ -9,3 +9,4 @@ http_proxies
 openapi
 duplicate_event
 binlog_parse
+foreign_key
```

Comment:
you can add this test at the beginning of this file, so CI will fail more earlier